runtime/rasdump/javadump.cpp

```diff
@@ -898,6 +997,15 @@ JavaCoreDumpWriter::writeProcessorSection(void)
 	_OutputStream.writeInteger(numberOfCpus, "%i");
 	_OutputStream.writeCharacters("\n");
 
+#if defined(J9OS_I5)
+        J9ProcessorDesc desc;
+        memset(&desc, 0, sizeof(J9ProcessorDesc));
+        (void)j9sysinfo_get_processor_description(&desc);
```

Comment:
This should handle the return value, not cast it away

Reply:
Searched all codes in eclipse/openj9.  Seems most of the case doesn't handle the return value.